h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -531,7 +533,8 @@ protected void checkMemoryFootPrint(DataInfo activeData) {
   }
 
   DataInfo _dinfo;
-
+  String[] _validPredictors;  // valid predictors are numerical columns only
```

Comment:
it would be better not to have an additional state in this class, the comment is confusing because it only makes sense in the context of VIF